src/binary/Yast.cc

```diff
@@ -319,10 +319,11 @@ static void start_config_console()
     // LoadError caught
     if (success == Qfalse) return;
 
-    // call "Installation::Console.run"
+    // call "Installation::Console::Menu.run"
     VALUE installation = rb_const_get(rb_cObject, rb_intern("Installation"));
     VALUE console = rb_const_get(installation, rb_intern("Console"));
-    rb_funcall(console, rb_intern("run"), 0);
+    VALUE menu = rb_const_get(console, rb_intern("Menu"));
+    rb_funcall(menu, rb_intern("run"), 0);
```

Comment:
please update documenation of method.

Reply:
Also what happens when exception is raised?

Reply:
Updated.

When an exception is raised then it is caught by the global exception handler (that "internal error" popup), just like with other code.